diff3proof/pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[build-system]
+requires = ["maturin>=1.0,<2.0"]
+build-backend = "maturin"
+
+[tool.maturin]
+# maturin needs a manifest with a [package], can't just use the workspace's Cargo.toml
```

Comment:
The inline comment is stale, unlike fontc you aren't placing a pyproject.toml next to the workspace's Cargo toml, here you simply have two distinct pyproject.toml each placed next to a non-work space Cargo.toml, there's probably no need for a custom manifest-path because it will pick the default Cargo.toml in the same folder.